layers.go

```diff
@@ -1329,6 +1342,17 @@ func (r *layerStore) PutAdditionalLayer(id string, parentLayer *Layer, names []s
 	return copyLayer(layer), nil
 }
 
+func pickStoreLocation(jsonPath *[numLayerLocationIndex]string, volatile, writeable bool) layerLocations {
```

Comment:
(Absolutely non-blocking: This could be a method on `*layerStore`)

Reply:
fixed and pushed a new version